packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -142,6 +142,9 @@ class _AnimatedState extends State<AnimatedWidget> {
   Widget build(BuildContext context) => widget.build(context);
 }
 
+/// Convenience function for passing around a builder for a transiton's secondary animation.
+typedef DelegatedTransitionBuilder = Widget Function(BuildContext context, Animation<double> animation, Animation<double> secondaryAnimation, Widget? child);
```

Comment:
Typedefs should mention at least one place where it is used: https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#leave-breadcrumbs-in-the-comments

Typedef docs should usually start with "Signature for": https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#canonical-terminology

"transiton" => "transition"